model/notification.go

```diff
@@ -0,0 +1,229 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+import (
+	"strings"
+	"unicode"
+	"unicode/utf8"
+
+	"github.com/mattermost/mattermost-server/v5/utils/markdown"
+)
+
+type ExplicitMentions struct {
+	// Mentions contains the ID of each user that was mentioned and how they were mentioned.
+	Mentions map[string]MentionType
+
+	// OtherPotentialMentions contains a list of strings that looked like mentions, but didn't have
+	// a corresponding keyword.
+	OtherPotentialMentions []string
```

Comment:
Same question here.

Reply:
The same answer here ;)

Reply:
`OtherPotentialMentions`is used in [app/notification.go](https://github.com/mattermost/mattermost-server/blob/3e3fcdf84502ac0a4652e1d6e0459e67ed5ac188/app/notification.go#L109) too

Reply:
Same as before, It's not a big change actually, I'll fix it in the next commit.